portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,89 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.Date;
+import java.text.DateFormat;
+import java.lang.Comparable;
+
+/** Represents a single comment from a user. */
+public class Comment implements Comparable<Comment> {
+  final private String text;
+  final private String name;
+  final private long timestamp;
+
+  public Comment(String text, String name) {
+    this.text = text;
+    this.name = name;
+    this.timestamp = System.currentTimeMillis();
```

Comment:
from a timestamp perspective, it's not generally a good idea to read directly from System.currentTimeMillis(), especially within models like this, for the following reasons: 
- When running a multi-region server setup, it doesn't account for timezones
- Tests are difficult to reproduce because the dependency on System is hardwired into it. 

Instead, for now, feel free to use System.currentTimeMillis() inside the HTTP server, and we'll show you alternatives in the future.

Reply:
I'm not sure I fully understand. According to the [Java docs],(https://docs.oracle.com/javase/8/docs/api/java/lang/System.html#currentTimeMillis--) `System.currentTimeMillis()` is in UTC.
Should I switch to a `Calendar` or a `Date` instead?

Reply:
oh, my apologies, you're right, it does have UTC. However, calling the method should still be avoided being called in constructors and such  - I'll add this to our 1:1 to discuss at a later time, since the impact is not Heavy in this case.